WoT-stats/src/app/service/api.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { PlayerId } from '../interfaces/playerId';
+import { PlayerStats } from '../interfaces/playerStats';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ApiService {
+  private apiUrl = 'https://api.worldoftanks.eu/wot/account/list/';
```

Comment:
Change to this  apiUrl = https://api.worldoftanks.eu/wot/account/ 
and use this variable below